bench-vortex/src/tpch/execute.rs

```diff
@@ -44,9 +43,11 @@ pub async fn run_tpch_query(
 
 pub async fn execute_query(ctx: &SessionContext, query: &str) -> Result<Vec<RecordBatch>> {
     let plan = ctx.sql(query).await?;
-    let (state, plan) = plan.into_parts();
-    let optimized = state.optimize(&plan)?;
-    let physical_plan = state.create_physical_plan(&optimized).await?;
-    let result = collect(physical_plan.clone(), state.task_ctx()).await?;
-    Ok(result)
+    plan.collect().await
```

Comment:
I thought this doesn't run all the optimizations steps

Reply:
yeah, commented it out when I was debugging some issue, I'll add it back in a second